pallets/msa/src/lib.rs

```diff
@@ -107,7 +107,7 @@ pub mod pallet {
 		UnauthorizedDelegate,
 		DelegateRevoked,
 		DelegateError,
-		DelegateNotFound,
+		DelegationNotFound,
```

Comment:
I renamed this because it was confusing me when it was the Delegator/Provider that was missing. This error really occurs when the delegation relationship itself is missing.